test/print_document_symbols.cc

```diff
@@ -48,12 +48,16 @@ pair<string, vector<string>> findEditsToApply(string_view filePath) {
     return make_pair(uri, move(fileContents));
 }
 
-int printDocumentSymbols(string_view filePath) {
+int printDocumentSymbols(char *paths[], int fileAmount) {
```

Comment:
I think I would slightly prefer a `std::vector<string_view>` -- shield the inner routine from the nastiness of argc/argv.

Reply:
That was my initial thought, but I couldn't find a decent way to create a `vector` from the `*char[]` without rolling a loop, so I decided to go with what I have